Complete contact form and add styles

BregornOriginal opened the pull request:
# Conctact form has been completed ❕

## Changes in this Section ℹ️

- Built a contact form in the footer
- Name field acepts up to 30 chars
- Email field accepts only emails
- Message field accepts up to 500 chars
- All fields have placeholder text
- All fields have the correct type
- A user cannot submit the form inf one or more of the fields is empty (for this used Formspree Service) 🕸️
- I have used Flexbox to complete this section and make it more responsible as possible
- Added interaction to buttons (enable, hover, disabled, pressed)▶️

Thank @biruk-bereka and @Justcoodeit, my coder's partners that help me to fix some 🐛.

biruk-bereka committed:
add description and form to contact form section

Justcoodeit committed:
added the button

BregornOriginal committed:
Complete contact form and add styles

BregornOriginal committed:
Correct color-hex-case

mahbubul14 commented on index.html:
```diff
@@ -1,290 +1,326 @@
+            </li>
+            <li>
+              <textarea name="message" maxlength="500" id="message" required>Enter your text here</textarea>
+            </li>
```
- [x] It would be better if you add a placeholder text here. :+1: 

Ex: `<textarea name="message" maxlength="500" id="message" placeholder="Enter your text here" required></textarea>`

BregornOriginal replied:
Thank for the comment, I will change that, you're right that looks better!

mahbubul14 commented on index.html:
```diff
@@ -1,290 +1,326 @@
+        </form>
+        <div class="container-button">
+          <button class="button-submit" type="submit"> Get in touch </button>
         </div>
```
- [x] The form submission of your page is not working. To fix that, you should include submit (Get in touch) button in the `form` tag right after the text area. :+1:

BregornOriginal replied:
Thank for the comment, I was missing that container outside of the form. Best!

mahbubul14 reviewed (changes requested):
Hi @BregornOriginal ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [x] In your `README` file, Kindly Update the proper content for the getting started section. This should be an example of how you may give instructions on setting up your project locally.

- [x] Please check the inline comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

BregornOriginal committed:
changed textarea and moved submit button

- Textarea now have a placeholder whit the request of the field
- The submit button now work correcly, it's was moved to the form container
- Also, the README file has a brief update to the work description

zieeco commented on index.html:
```diff
@@ -289,14 +289,14 @@ <h5 class="title-recent-works-one">Multi-Post Stories</h5>
               <input type="email" name="email_address" id="email-address" placeholder="Email address" required>
```
- [ ] **[OPTIONAL]**

- Please, kindly fix this issue of width with your inputs
![input](https://user-images.githubusercontent.com/59397159/165435823-8c01cff3-e8c7-4400-8624-418745e39b20.png)

BregornOriginal replied:
@zieeco Thank you for the review, I will fix that now! Excellent view.

zieeco reviewed (approved):
# status: APPROVED :green_circle: :100: 

Hi @Team


![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966](https://user-images.githubusercontent.com/85113567/164945882-0a9554ea-956c-4fa0-a95c-379285283c1d.gif)











Good job so far on implementing the changes required by the previous reviewers 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️






## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
